components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -670,12 +670,21 @@ public DataSection familyHistoryBody(Patient patient)
         PatientData<List<String>> ethnicities = patient.getData("ethnicity");
         Integer x = 0;
         if (present.contains("global_mode_of_inheritance")) {
-            PatientData<SolrOntologyTerm> globalControllers = patient.<SolrOntologyTerm>getData("global-qualifiers");
-            SolrOntologyTerm modeTerm =
+            PatientData<List<SolrOntologyTerm>> globalControllers = patient.getData("global-qualifiers");
+            List<SolrOntologyTerm> modeTermList =
                 globalControllers != null ? globalControllers.get("global_mode_of_inheritance") : null;
-            String mode = modeTerm != null ? modeTerm.getName() : "";
-            DataCell cell = new DataCell(mode, x, 0);
-            bodySection.addCell(cell);
+            int y = 0;
+            if (modeTermList != null) {
```

Comment:
This should probably be `modeTermList != null && !modeTermList.isEmpty`. Otherwise formatting will be off.